### PR TITLE
Wrap Additional v3.3 API

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by musm** [comment]
Let me finish up some final v3.3 API  then let you know !

**Comment by musm** [comment]
This is good to go on my end!

File: src/glfw3.jl [hunk 1]
```diff
@@ -601,6 +616,7 @@ GetWindowAttrib(window::Window, attrib::Integer) = ccall((:glfwGetWindowAttrib,
 @windowcallback WindowIconify(window::Window, iconified::Cint) -> (window, Bool(iconified))
 @windowcallback FramebufferSize(window::Window, width::Cint, height::Cint)
 @windowcallback WindowContentScale(window::Window, xscale::Cfloat, yscale::Cfloat)
+@windowcallback WindowMaximize(window::Window, maximized::Cint)
```
Comment (jayschwa) [inline src/glfw3.jl:619]:
```suggestion
@windowcallback WindowMaximize(window::Window, maximized::Cint) -> (window, Bool(maximized))
```

**Comment by musm** [comment]
thanks

**Pull request merged**